examples/acados_python/linear_mass_model/linear_mass_test_problem.py

```diff
@@ -164,7 +164,7 @@ def solve_maratos_ocp(setting):
     # ocp.solver_options.print_level = 1
     ocp.solver_options.nlp_solver_type = 'SQP' # SQP_RTI, SQP
     ocp.solver_options.globalization = globalization
-    ocp.solver_options.alpha_min = 0.01
+    ocp.solver_options.globalization_alpha_min = 0.01
```

Comment:
I would also add a test with the deprecated options, after we address this part in the Python options.

Reply:
I added this test